developer/src/kmcmplib/tests/get-test-source.sh

```diff
@@ -9,5 +9,8 @@
 set -eu
 find "$1" -name '*.kmn' | \
   grep -E '(release|experimental)/([a-z0-9_]+)/([a-z0-9_]+)/source/\3\.kmn$' | \
-  grep -v masaram_gondi
+  grep -vE 'masaram_gondi|anii|sil_kmhmu|fv_statimcets|fv_nuucaanul'
 # #12623: exclude masaram_gondi due to #11806
+# #12631: exclude anii, sil_kmhmu as ico references have mismatching case
+# #12631: exclude fv_statimcets, fv_nuucaanul as these include U+2002 which is not
+#         treated as whitespace on mac arch
```

Comment:
Can you copy this summary in the PR description?
 
 To make more visible on seeing why more keyboards are excluded.

And what about making this list a BASH constant? (maintainability)

Reply:
> Can you copy this summary in the PR description?
> 
> To make more visible on seeing why more keyboards are excluded.

Added all the details to the PR description.

> And what about making this list a BASH constant? (maintainability)

I don't think that buys us anything -- we've got the comment and it's a single string, not repeated anywhere.